config/initializers/devise.rb

```diff
@@ -127,7 +127,7 @@
   # initial account confirmation) to be applied. Requires additional unconfirmed_email
   # db field (see migrations). Until confirmed, new email is stored in
   # unconfirmed_email column, and copied to email column on successful confirmation.
-  config.reconfirmable = true
+  config.reconfirmable = false
```

Comment:
Oo I don't think we want to turn that off globally, just for the case where the admin makes a change to a user.